Makefile

```diff
@@ -14,7 +14,7 @@ RDS_VERSION ?= 0.0.27
 ORG ?= ecosystem-appeng
 
 # CATALOG_BASE_IMG defines an existing catalog version to build on & add bundles to
-#CATALOG_BASE_IMG ?= quay.io/$(ORG)/rds-dbaas-operator-catalog:v$(VERSION)
+CATALOG_BASE_IMG ?= quay.io/ecosystem-appeng/rds-dbaas-operator-catalog:v0.1.0
```

Comment:
why sticking with v0.1.0 ?

Reply:
because that's the prior release we're going to build this release on top of

Reply:
I see the operator docker image version is different nvm